code/cwe/RenderCsvData.hs

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env cabal
+{- cabal:
+build-depends: base, csv
+-}
+-- | Use this script to update the CWE.Raw module:
+-- Go to https://cwe.mitre.org/data/downloads.html
+-- Download and extract the 'Software Development' and 'CWE Simplified Mapping' CSV.zip files
+-- Run the following command: ./RenderCsvData.hs | fourmolu --stdin-input-file ./src/CWE/Raw.hs > src/CWE/Raw.hs
+module Main where
+
+import Data.List
+import Data.Maybe
+import Text.CSV
+import Text.Read
+
+main :: IO ()
+main = do
+    dbs <- traverse readCSV ["699.csv", "1003.csv"]
```

Comment:
These seem to be CVE views, which are incomplete.
I think we want the full list, which is at the top of the download page.  Especially because we will publish this library for others to use.  Unfortunately there does not seem to be CSV artifact for the full list - only the XML.